### PR TITLE
fix(suite-native): customizable qrcode background elevation

### PR DESCRIPTION
**Pull request by PeKne**
The QRCode component `backgroundElevation` prop added to make the buttons visible on a every elevation.
Screenshot 2023-03-06 at 12 12 13

### CODE REVIEW

**Comment by PeKne** [comment]
@mnuky This fixes the problem with the "invisible" button background.

**Comment by Nodonisko** [comment]
Can you please also add something like 30px border around QR code itself that will be always white in all themes? This QR code is nearly impossible to scan in dark mode

**Comment by PeKne** [comment]
Dark mode Qrcode border fixed in 7b65ae46 :
Screenshot 2023-03-06 at 14 11 17

@Nodonisko

File: suite-native/qr-code/src/components/QRCode.tsx [hunk 1]
```diff
@@ -58,19 +64,26 @@ export const QRCode = ({
...
-                    <ReactQRCode
-                        bgColor={colorVariants.standard.backgroundSurfaceElevation1}
-                        fgColor={colorVariants.standard.backgroundNeutralBold}
-                        level="Q"
-                        size={QRCODE_SIZE}
-                        value={data}
-                    />
+                    <View
+                        style={applyStyle(qrCodeContainerStyle, {
+                            backgroundColor: colorVariants.standard.backgroundSurfaceElevation1,
```
Comment (Nodonisko) [inline suite-native/qr-code/src/components/QRCode.tsx]: Why are you passing it as style prop? Why not just place it to style?

**Review by juriczech — APPROVED** [review]

File: suite-native/qr-code/src/components/QRCode.tsx (hunk 1 above)
Comment (PeKne) [reply suite-native/qr-code/src/components/QRCode.tsx]: To make the function pure. There is basically no difference between my and your solution, but I believe that the style function should take input only through its arguments. Should I change it anyway?

Comment (Nodonisko) [reply suite-native/qr-code/src/components/QRCode.tsx]: Function will be pure even if it's used directly because colorVariants is constant.

Comment (PeKne) [reply suite-native/qr-code/src/components/QRCode.tsx]: fixed ba9fbe6c7

**Review by Nodonisko — APPROVED** [review]

**Pull request merged**